### PR TITLE
Add SQS FIFO high throughput options

### PR DESCRIPTION
**Pull request by redradrat**
##### SUMMARY
adds high throughput options for SQS fifo queues

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

* sqs_topic

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/sqs_queue.py [hunk 1]
```diff
@@ -39,6 +39,20 @@
...
+    choices: ['queue', 'messageGroup']
+    default: 'queue'
+    type: str
+  fifo_throughput_limit:
+    description:
+      - Throughput limit for FIFO queues.
+      - C(perMessageGroupId) is required for high throughput FIFO.
+    choices: ['perQueue', 'perMessageGroupId']
+    default: 'perQueue'
+    type: str
```
Comment (alinabuzachis) [inline plugins/modules/sqs_queue.py:55]: Please add `version_added: 5.1.0`

File: plugins/modules/sqs_queue.py [hunk 2]
```diff
@@ -100,6 +114,16 @@
...
+fifo_throughput_limit:
+    description: Which throughput limit strategy is applied.
+    type: str
+    returned: always
+    sample: perQueue
+deduplication_scope:
+    description: The deduplication setting.
+    type: str
+    returned: always
+    sample: messageGroup
```
Comment (alinabuzachis) [inline plugins/modules/sqs_queue.py:126]: Please add `version_added: 5.1.0`

File: plugins/modules/sqs_queue.py [hunk 3]
```diff
@@ -100,6 +114,16 @@
     type: bool
     returned: always
     sample: True
+fifo_throughput_limit:
+    description: Which throughput limit strategy is applied.
+    type: str
+    returned: always
+    sample: perQueue
```
[the remark this answers is not in the document]
Comment (tremble) [reply plugins/modules/sqs_queue.py:123]: already existed, just documented

File: plugins/modules/sqs_queue.py (hunk 2 above)
Comment (tremble) [reply plugins/modules/sqs_queue.py:128]: already existed, just documented

**Review by tremble — APPROVED** [review]

**Pull request merged**